Orange/widgets/widget.py

```diff
@@ -114,6 +114,10 @@ class OWWidget(QDialog, Report, metaclass=WidgetMetaClass):
     #: area to the right of the `controlArea`).
     want_main_area = True
     #: Should the widget construct a `controlArea`.
+    want_standard_buttons_box = "vertical"
```

Comment:
How about just setting this to default `'horizontal'`? Report and Save image both fit in one line. :smiley: